src/main/scala/org/broadinstitute/dsde/workbench/sam/service/UserService.scala

```diff
@@ -172,9 +172,9 @@ class UserService(val directoryDAO: DirectoryDAO, val cloudExtensions: CloudExte
     directoryDAO.loadUser(userId, samRequestContext).flatMap {
       case Some(user) =>
         for {
-          ldapStatus <- registrationDAO.isEnabled(user.id, samRequestContext)
+          tosStatus <- getTermsOfServiceStatus(user.id, samRequestContext)
           adminEnabled <- directoryDAO.isEnabled(user.id, samRequestContext)
-        } yield Some(UserStatusInfo(user.id.value, user.email.value, ldapStatus, adminEnabled))
+        } yield Some(UserStatusInfo(user.id.value, user.email.value, tosStatus.getOrElse(true) && adminEnabled, adminEnabled))
```

Comment:
if `tosStatus` is None, we default it to `true`?

This seems to require all users to be `adminEnabled`?

Reply:
Yes to the first because if tosStatus is None that means tos is disabled and should be ignored

Yes to the second, `adminEnabled` means they have been enabled by an admin, or rather, have not been disabled by an admin. All new users start our with this true.